localstack-core/localstack/services/kms/provider.py

```diff
@@ -368,6 +370,9 @@ def _is_rsa_spec(key_spec: str) -> bool:
     #
 
     @handler("CreateKey", expand=False)
+    @kms_feature.Provisioning
+    @kms_api.CreateKey
+    @ApiLimitation(limitation="Status 'Updating' is not supported.")
```

Comment:
testing both approaches here:
* the second one `ApiLimitation` would be more generic, eg can be reused for every operation, and we can provide the actual limitations as input (currently string, but we could change this to a list as well).
  * if there are a lot of limitations we could use constants that are defined in a separate file, to not pollute the code too much
  * tested this approach in the `playground.py` script, e.g. it will be added as `api_limitation`:
  ```
          "Provisioning": {
          "docs": {
            "general_docs": "Manage the creation and modification of the keys.",
            "implementation_status": "ImplementationStatus.FULLY_IMPLEMENTED",
            "support_type": "SupportStatus.SUPPORTED",
            "aws_docs_link": "https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html",
            "api_operations": [
              "CreateKey"
            ]
          },
          "operations": [
            "localstack.services.kms.provider.KmsProvider.create_key",
            "localstack.services.kms.provider.KmsProvider.update_key_description"
          ],
          "api_limitations": {
            "localstack.services.kms.provider.KmsProvider.create_key": "Status 'Updating' is not supported."
          }
        },
   ```